webapp-django/crashstats/manage/jinja2/manage/reprocessing.html

```diff
@@ -0,0 +1,40 @@
+{% extends "manage/base.html" %}
+
+{% block page_title %}{{ super() }} - Re-processing{% endblock %}
+
+{% block admin_title %}{{ super() }} - Re-processing{% endblock %}
+
+{% block mainbody %}
+<div class="panel">
+    <div class="body notitle">
+
+        <h3>
+            Put a Crash ID back in for re-processing
+        </h3>
+
+        {% if crash_id %}
+        <p>
+            <a href="{{ url('crashstats:report_index', crash_id) }}">
+                Report Index for
+                <code>{{ crash_id }}</code>
+            </a>
+        </p>
+        {% endif %}
+
+        <form action="" method="post">
+            {% csrf_token %}
+            <table class="data-table">
+                {{ form }}
+                <tr>
+                    <th>&nbsp;</th>
+                    <td>
+                        <input type="submit" value="Let's do this!">
```

Comment:
I know it's bikeshedding because that's the admin panel and there's only a handful of people using it, but it's generally a good idea to keep button labels very explicit.